picard/pluginmanager.py

```diff
@@ -278,11 +285,11 @@ def _load_plugin_from_directory(self, name, plugindir):
                           "version of Picard.") % (plugin.name, plugin.file)
                 self.plugin_error(plugin.name, error, log_func=log.warning)
         except VersionError as e:
-            error = _("Plugin %r has an invalid API version string : %s") % (name, e)
-            self.plugin_error(name, error)
+            error = _("Plugin %r has an invalid API version string : %s")
+            self.plugin_error(name, error, name, e)
```

Comment:
Why pass `e` here instead of using `log.exception`?

Reply:
No, you are right. No need to clutter the output with a traceback here, the exception and message are clear enougj